docs/source/markdown/podman-build.1.md

```diff
@@ -86,7 +86,7 @@ resulting image's configuration.
 #### **--cache-from**
 
 Images to utilize as potential cache sources. Podman does not currently support
-caching so this is a NOOP.
+caching so this is a NOOP. (Not available for remote commands)
```

Comment:
Suggest "This option is not available with the remote Podman client"

Reply:
We have this comment all over the podman-run and podman-create man page, so I will make the change everywhere.  Want to get @TomSweeneyRedHat agreement though so we don't bikeshed.

Reply:
@mheon's suggestion SGTM